tfjs-backend-webgpu/src/kernels/clip_webgpu.ts

```diff
@@ -25,25 +25,21 @@ export class ClipProgram implements WebGPUProgram {
   outputShape: number[];
   shaderKey: string;
   variableNames = ['A'];
+  uniforms = 'float minVal; float maxVal;';
   dispatchLayout: {x: number[]};
   dispatch: [number, number, number];
-  workPerThread = 1;
   workGroupSize: [number, number, number] = [64, 1, 1];
   minVal: number;
   maxVal: number;
   size: number;
 
-  constructor(outputShape: number[], minVal: number, maxVal: number) {
+  constructor(outputShape: number[]) {
     this.outputShape = outputShape;
     this.dispatchLayout = flatDispatchLayout(this.outputShape);
     this.dispatch = computeDispatch(
-        this.dispatchLayout, this.outputShape, this.workGroupSize,
-        [this.workPerThread, 1, 1]);
+        this.dispatchLayout, this.outputShape, this.workGroupSize, [1, 1, 1]);
```

Comment:
nit: ` this.dispatch = computeDispatch(this.dispatchLayout, this.outputShape, this.workGroupSize);`

Reply:
done